Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -121,22 +123,33 @@ const StatusBar = React.createClass({
     },
 
     setBackgroundColor(color, animated?: boolean) {
-      if (Platform.OS !== 'android') {
-        console.warn('`setBackgroundColor` is only available on Android');
+      if (Platform.OS === 'ios') {
+        console.warn('`setBackgroundColor` is not available on iOS');
         return;
       }
-      animated = animated || false;
-      StatusBar._defaultProps.backgroundColor = color;
-      StatusBarManager.setColor(processColor(color), animated);
+      else
```

Comment:
I still don't like this (sorry!).  I want to make it very explicit what belongs to windows vs. ios vs. android.  Make sure the outer structure of the conditional looks like:
if (Platform.OS === 'ios') {
   ...
}  else if (Platform.OS === 'android') {
   ...
} else if (Platform.OS === 'windows') {
   ...
}